CRM/Core/Smarty/plugins/function.crmURL.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC
+ *
+ */
+
+/**
+ * Generate a url
+ *
+ * @param string $string
+ *   The html to be tweaked with.
+ *
+ * @return string
+ */
+function smarty_function_crmURL($string, $params) {
+  return CRM_Utils_System::url();
```

Comment:
If that passes, it says more about the inadequacy of our test suite...

Reply:
yeah I didn't want to overdo it